int16 inflatable_state_flag
